THIRD-PARTY-LICENSES.csv

```diff
@@ -618,6 +617,7 @@
 "globby@6.1.0","MIT","https://github.com/sindresorhus/globby"
 "graceful-fs@4.1.11","ISC","https://github.com/isaacs/node-graceful-fs"
 "graceful-fs@4.2.6","ISC","https://github.com/isaacs/node-graceful-fs"
+"graceful-fs@4.2.8","ISC","https://github.com/isaacs/node-graceful-fs"
```

Comment:
I guess we have two of these due to two different dependencies using different versions?

Reply:
yup:
```
dockstore-ui2@2.8.0 /home/gluu/dockstore-ui2
├─┬ @angular-devkit/build-angular@12.0.4
│ ├─┬ @ngtools/webpack@12.0.4
│ │ └─┬ enhanced-resolve@5.7.0
│ │   └── graceful-fs@4.2.6
│ ├─┬ less@4.1.1
│ │ └── graceful-fs@4.1.11
│ ├─┬ webpack@5.38.1
│ │ ├─┬ enhanced-resolve@5.8.2
│ │ │ └── graceful-fs@4.2.6  deduped
│ │ ├── graceful-fs@4.2.6
│ │ └─┬ watchpack@2.2.0
│ │   └── graceful-fs@4.1.11  deduped
│ └─┬ webpack-dev-server@3.11.2
│   └─┬ chokidar@2.1.8
│     └─┬ readdirp@2.2.1
│       └── graceful-fs@4.1.11  deduped
├─┬ @angular/cli@12.0.4
│ └─┬ pacote@11.3.2
│   └─┬ @npmcli/run-script@1.8.5
│     └─┬ node-gyp@7.1.2
│       └── graceful-fs@4.2.6
├─┬ @datorama/akita@6.2.3
│ └─┬ schematics-utilities@2.0.3
│   └─┬ @schematics/update@0.803.29
│     └─┬ pacote@9.5.5
│       └─┬ cacache@12.0.4
│         ├── graceful-fs@4.2.8
│         └─┬ move-concurrently@1.0.1
│           └─┬ fs-write-stream-atomic@1.0.10
│             └── graceful-fs@4.1.11  deduped
├─┬ cypress@8.0.0
│ └─┬ fs-extra@9.1.0
│   ├── graceful-fs@4.2.6
│   └─┬ jsonfile@6.1.0
│     └── graceful-fs@4.2.6  deduped
├─┬ karma@6.3.4
│ ├── graceful-fs@4.2.6
│ └─┬ log4js@6.3.0
│   └─┬ streamroller@2.2.4
│     └─┬ fs-extra@8.1.0
│       ├── graceful-fs@4.2.6
│       └─┬ jsonfile@4.0.0
│         └── graceful-fs@4.1.11  deduped
├─┬ license-checker@25.0.1
│ └─┬ read-installed@4.0.3
│   ├── graceful-fs@4.1.11  deduped
│   ├─┬ read-package-json@2.1.1
│   │ └── graceful-fs@4.1.11  deduped
│   └─┬ readdir-scoped-modules@1.1.0
│     └── graceful-fs@4.1.11  deduped
└─┬ ts-loader@9.2.3
  └─┬ enhanced-resolve@5.8.2
    └── graceful-fs@4.2.6
```